app/models/User.js

```diff
@@ -170,6 +170,10 @@ module.exports = (User = (function () {
     isSmokeTestUser () { return User.isSmokeTestUser(this.attributes) }
     isIndividualUser () { return !this.isStudent() && !User.isTeacher(this.attributes) }
 
+    getSubscriptionLevel () {
+      return 'free'
```

Comment:
need to modify this. On vue, we can use this: https://github.com/codecombat/codecombat/blob/master/app/core/store/modules/me.js#L61